prisma/schema.sql

```diff
@@ -58,7 +58,7 @@ CREATE TYPE ApplicationStatus as ENUM(
 );
 
 -- Create reason for replacement enum
-CREATE TYPE ReasonForReplacement as ENUM('LOST', 'STOLEN', 'OTHER');
+CREATE TYPE ReasonForReplacement as ENUM('LOST', 'STOLEN', 'MAIL_LOST', 'OTHER');
```

Comment:
We'll need to run a migration to make this change in the existing production database. That could be risky though, given this project hasn't been initialized to use prisma migrate yet. I suggest holding off on this feature.

https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/add-prisma-migrate-to-a-project

Reply:
This makes sense, closing for now!